api/src/pcapi/core/external_bookings/cds/client.py

```diff
@@ -94,7 +96,17 @@ def get_venue_movies(self) -> list[external_bookings_models.Movie]:
         return [cds_movie.to_generic_movie() for cds_movie in cds_movies]
 
     def get_movie_poster(self, image_url: str) -> bytes:
-        return get_movie_poster_from_api(image_url)
+        try:
+            return get_movie_poster_from_api(image_url)
+        except:
```

Comment:
```suggestion
        except CineDigitalServiceAPIException:
``` 
?

pareil pour EMS

Reply:
J'aurais dû mettre cette pull request en "draft", je n''avais pas terminé. ;) 

C'est fait, j'ai mis les bonnes exceptions à chaque fois.

Reply:
ahah désolé